homeassistant/components/rointe/const.py

```diff
@@ -0,0 +1,44 @@
+"""Constants for the Rointe Heaters integration."""
+
+import logging
+
+from homeassistant.backports.enum import StrEnum
+from homeassistant.const import Platform
+
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "rointe"
+DEVICE_DOMAIN = "climate"
+PLATFORMS: list[Platform] = [Platform.CLIMATE, Platform.SENSOR, Platform.UPDATE]
+CONF_USERNAME = "rointe_username"
+CONF_PASSWORD = "rointe_password"
+CONF_INSTALLATION = "rointe_installation"
+
+ROINTE_MANUFACTURER = "Rointe"
+
+ROINTE_SUPPORTED_DEVICES = ["radiator", "towel", "therm", "radiatorb"]
+
+CMD_SET_TEMP = "cmd_set_temp"
+CMD_SET_PRESET = "cmd_set_preset"
+CMD_HVAC_OFF = "cmd_turn_off"
+CMD_SET_HVAC_MODE = "cmd_set_hvac_mode"
```

Comment:
Since these are options of a defined set, should they be enums (to be consistent elsewhere)?